contrib/oid2name/meson.build

```diff
@@ -17,6 +17,10 @@ oid2name = executable('oid2name',
 )
 contrib_targets += oid2name
 
+# if llvm.found()
```

Comment:
Commented out code

Reply:
I forgot to remove, I noticed this target doesn't have bitcode generated in autoconf setup, it links to frontend libraries and seems not required to have bitcode:
```
oid2name = executable('oid2name',
  oid2name_sources,
  dependencies: [frontend_code, libpq],
  kwargs: default_bin_args,
)
```
fixup: f59f98e5c5d4ea3a0e47078efa9e4a08e757ee71